lib/hooks/useSendTransaction.js

```diff
@@ -1,16 +1,13 @@
-import { useCallback, useContext } from 'react'
+import { useCallback } from 'react'
+import { useOnboard } from '@pooltogether/hooks'
 
-import { WalletContext } from 'lib/components/WalletContextProvider'
 import { useNetwork } from 'lib/hooks/useNetwork'
```

Comment:
I wonder if `useNetwork` and `useWalletNetwork` should also go in `@pooltogether/hooks` - or just be removed in favour of the `useOnboard` hook.